### PR TITLE
Add v3-8×1.8b-exp1

### PR DESCRIPTION
**Pull request by Taishi-N324**

### CODE REVIEW

File: models/v3-8×1.8b-exp1.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+|Spec name|Value|
+|:---|:---|
+|Data center|TSUBAME 4.0|
+|Number of Nodes|16|
+|CPU|AMD EPYC 9654|
+|Number of CPUs per instance|2|
+|RAM|768 GiB|
+|Accelerators| NVIDIA H100 SXM5 94GB HBM2e |
+|Number of accelerators per instance|4|
+|Intra Node Communication |NVLink|
```
Comment (odashi) [inline models/v3-8×1.8b-exp1.md:67]: Sakuraの設定に書き換えておいてください。

Comment (Taishi-N324) [reply models/v3-8×1.8b-exp1.md]: https://github.com/llm-jp/model-cards/pull/25/commits/6cb4cdacbd06a13bd2bbf51758e22aad6bee5baa
修正しました。

File: models/v3-8×1.8b-exp1.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+|Beta1|0.9|
+|Beta2|0.95|
+|Epsilon|1e-8|
+|Weight decay factor|0.1|
+|Warmup strategy|Linear|
+|Warmup steps|2000|
+|Learning rate scheduling strategy|Cosine|
+|Learning rate scheduling steps|492120|
+|Total training steps|494120|
+|Global batch size|1,024|
```
[the remark this answers is not in the document]
Comment (Taishi-N324) [reply models/v3-8×1.8b-exp1.md]: 1,024にしようと思います。
https://arxiv.org/abs/2410.07524 
最近出たupcycle関連論文でバッチサイズはある程度大きい方がいいと言っていることもあります
またDeepseek-V2なども大きなバッチサイズを採用していたりします
Deepseek-V2 [8] also used a large batch size of 9216 (more than 37M tokens) and learning rate of 2.4e-4 to pretrain MoE models.

File: models/v3-8x1.8b-exp1.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+|Processor|Intel Xeon Platinum 8480|
+|Number of CPUs per instance|2|
+|RAM|2.0TB|
+|Accelerators|NVIDIA H100 80GB|
+|Number of accelerators per instance|8|
+|Intra Node Communication |NVLink|
+
+# Comments
+
+学習データはGENIAC 172Bモデルと同様であるため、GENIAモデルのモデルサイズ縮小版としての性格をもちます。
```
[the remark this answers is not in the document]
Comment (Taishi-N324) [reply models/v3-8x1.8b-exp1.md]: https://github.com/llm-jp/model-cards/pull/25/commits/b981ef6fc130122cfccaf446ee48bae08012fe9a
削除

[the remark this answers is not in the document]
Comment (Taishi-N324) [reply models/v3-8x1.8b-exp1.md]: 8x3.8Bも対応 https://github.com/llm-jp/model-cards/pull/25/commits/1127760256bdd087a7562b7da2f700282cf380c0

**Review by odashi — COMMENTED** [review]
Anonymityの破壊要素があるので、一時的にリポジトリをprivateに変更します。
